src/helics/apps/helics-broker.cpp

```diff
@@ -47,12 +51,24 @@ int main(int argc, char* argv[])  // NOLINT
         autorestart,
         "helics-broker --autorestart <broker args ...> will start a continually regenerating broker "
         "there is a 3 second countdown on broker completion to halt the program via ctrl-C\n");
-    cmdLine.add_flag("--http",
-                     http_webserver,
-                     "start an http webserver that can respond to queries on the broker");
-    cmdLine.add_flag("--web",
+
+    cmdLine.add_flag("--http,--web",
+                     http_server,
+                     "start a webserver to respond to http rest api requests");
+    cmdLine.add_flag("--websocket",
```

Comment:
From around line 930 in src/helics/apps/helicsWebServer.cpp it looks like only one of these two options should be given -- does CLI11 have a way to specify that these two options shouldn't be given at the same time? Or do we want to update the description for the `--http,--web` to mention that it will be ignored if `--websocket` is also given?

Reply:
CLI11 can do that,  I will look into that more

Reply:
They are not mutually exclusive, the code around 930 is in a class  that just needed a simple way to discriminate between http_server and websocket server.    I moved the default port so they now don't conflict by default.